docs/pages/setup/admin/daemon.mdx

```diff
@@ -33,89 +36,106 @@ $ readlink /sbin/init
 
 ## Step 1/3. Install and configure Teleport
 
-Choose the appropriate instructions for your environment.
+If you have not already, select and complete the appropriate installation instructions for your environment.
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Teleport requires a configuration file to run. After installation, no
-configuration file exists. We will create a minimal configuration file to show
-you how to run Teleport as a daemon:
+Teleport requires a configuration YAML file to be created and configured in order to run. After initial installation, no
+configuration file exists until we create it. This guide will use a minimal teleport configuration based on defaults, however in production environments we
+recommend reviewing configuration file options to better suit your own specific needs.
+
+To create the default teleport configuration file, enter the following command:
 
 ```code
 $ sudo teleport configure -o file
 Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
 ```
 
-## Step 2/3. Create a systemd unit file
+## Step 2/3. Creating and Configuring the systemd Daemon
 
-Copy the recommended Teleport unit file for `systemd` below and paste the
-contents into a file called `/usr/lib/systemd/system/teleport.service`:
+Once you've initially installed the `teleport` binary and created a Teleport configuration file, let's create and configure the teleport daemon.
 
-```systemd
-[Unit]
-Description=Teleport SSH Service
-After=network.target
+Systemd is designed to use **unit files**, which are the primary file used to define and set the conditions for
```

Comment:
Reworded this a bit. The path of the configuration file helps defines permissions and load order when used as intended. The wording in this paragraph was intended as a nod to that, but I'm avoiding going into greater length since I think that explainer is a little outside of the scope of the issue we're trying to resolve in this guide and could instead cause confusion. I worded this mostly per your recommendations because I think confusion is a risk otherwise. Resolving for now, but let me know if you have additional feedback!